README.md

```diff
@@ -163,57 +180,28 @@ and select the profiles you want to build.
 
-To correctly setup the environment, proceed as follows:
-- Install a jdk-8 distribution like [Eclipse Temurin](https://adoptium.net/temurin/releases/?version=8) for your specific CPU architecture and OS.
-- Start the Eclipse Installer
-- Switch to advanced mode (top right hamburger menu > Advanced Mode)
-- Select "Eclipse IDE for Eclipse Committers" and configure the "Product Version" to be the version **2023-03 or newer**.
-- Select the Eclipse Kura™ installer from the list. If this is not available, add a new installer from https://raw.githubusercontent.com/eclipse/kura/develop/kura/setups/kura.setup, then check and press the Next button
-- Select the "Developer Type":
-  - "User": if you want to develop applications or bundles running on Kura™, select this option. It will install only the APIs and the examples.
-  - "Developer" : if you are a framework developer, select this option. It will download and configure the Eclipse Kura™ framework.
-- Update Eclipse Kura™ Git repository username and customize further settings if you like (e.g. Root install folder, Installation folder name). To show these options, make sure that the "Show all variables" checkbox is enabled
-- Set the `JRE 1.8 location` value to the installed local jdk-8 VM
-- Leave all Bootstrap Tasks selected and press the Finish button
-- Accept all the licenses and wait for the installation to finish
-At first startup Eclipse IDE will checkout the code, perform a full build and configure a few Working Sets
-- When the tasks are completed. In the IDE open (double click) Target Platform > Target-Definition > Kura™ Target Platform Equinox 3.16.0, and press "Set as Target Platform" located at the top right of the window
-
-Now you are ready to develop on Eclipse Kura™.
-
-To raise an issue, please report a bug on [GitHub issues](https://github.com/eclipse/kura/issues/new).
+After this command runs, images can be found in your preferred container engine image list.
 
-### Known Issues
-Currently, the emulator web ui is not properly working on Windows so, with your setup, you will be able to build and deploy you applications, but not be able to use the Eclipse IDE based Kura™ emulator.
+IDE Setups
+----------
 
-The full build of Kura™ is only supported for Linux and Mac Os based systems.
+We currently support two setups for Kura™ development:
 
-Currently the maven build on Windows requires to disable the tests and will fail when it tries to create the installers for the target platforms.
+- [**Kura™ Development Environment Setup**](https://eclipse.github.io/kura/latest/java-application-development/development-environment-setup/): This is the full setup allowing you to contribute to the core Kura™ project codebase. It will install all the IDE plugins and formatters to have a pleasant development experience and clone the Kura™ source code on your workstation.
```

Comment:
Eclipse Kura

Reply:
Addressed in: 3fafedaed27d31d091bd64e67c809a32d5e77ac3